lib/ansible/modules/system/pid.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: pid
+version_added: 2.8
+description: "Gives the pid of the given process name"
```

Comment:
This should mention that the result is a list of none, one, or more PIDs. Also, please mention both "process ID" and "PID" so that readers who know only one of those terms understand that they mean the same.